docs/source/reference/changelog.md

```diff
@@ -20,6 +20,158 @@ Contributors to major version bumps in JupyterHub include:
 
 ## [Unreleased]
 
+## 5.0
+
+### 5.0.0 - 2024
+
+5.0.0 is a major release of JupyterHub.
+It has lots of cool new features.
```

Comment:
Maybe we can have the title reference "beta" somewhere for now, and perhaps lead with a note on how the changelog entry is planned to be updated or not over time as we release another beta or 5.0.0 stable.

Reply:
Good call. Done